ecosystem/sep-0005.md

```diff
@@ -96,6 +105,16 @@ m/44'/148'/8' GDJTCF62UUYSAFAVIXHPRBR4AUZV6NYJR75INVDXLLRZLZQ62S44443R SCD5OSHUU
 m/44'/148'/9' GBTVYYDIYWGUQUTKX6ZMLGSZGMTESJYJKJWAATGZGITA25ZB6T5REF44 SCJGVMJ66WAUHQHNLMWDFGY2E72QKSI3XGSBYV6BANDFUFE7VY4XNXXR
 ```
 
+Shamir Secret Sharing shares (non-deterministic):
```

Comment:
I should add that this is a 2-3 scheme so people don't have to struggle with guessing even if that's the most common thing.